src/ui/export-popup.vue

```diff
@@ -64,15 +64,29 @@
               </select>
             </div>
           </div>
+          <div class="form-group row">
+            <label
+              class="col-sm-3 col-form-label"
+              for="export_format"
+            >mit Analyse</label>
+            <div class="col-sm-7">
+              <input
+                id="exportAnalyzeCheck"
+                v-model="exportAnalyze"
+                type="checkbox"
+                class="form-control-sm"
+              >
+            </div>
+          </div>
         </div>
         <div class="modal-footer">
           <button
             class="btn btn-primary"
-            :disabled="!pdfReady && selectedFileformat === 'pdf'"
+            :disabled="!pdfReady && selectedFileformat === 'pdf' && exportAnalyze"
             @click="doExport"
           >
             <span
-              v-if="pdfReady || selectedFileformat !== 'pdf'"
+              v-if="pdfReady || selectedFileformat !== 'pdf' || !exportAnalyze"
```

Comment:
I don't fully understand this button logic, but does it really make sense to include the `exportAnalyze` in the condition here?

Reply:
It does make sense, but it would also work without it.
The button is only disabled in case we need the analysis table.
The analysis table is only needed in case we want to export a pdf.
=> it is always clickable if selectedFileformat !== 'pdf'
If we want to export a pdf we need to wait for the table to finish loading => pdfReady === true
But if we want to export a pdf without the analysis table we dont need to wait for it => exportAnalyze === false -> we can ignore pdfReady.

I will rename pdfReady to AnalyzeTableReady to make this clearer.